content/blog/from-generative-to-agentic-ai-tracing-the-leap-from-words-to-actions.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "From Generative to Agentic AI: Tracing the Leap from Words to Actions"
+date: 2025-07-03T10:33:21.361Z
+author: DINESH R SINGH
+authorimage: /img/Avatar1.svg
```

Comment:
If you have an author image (size: 192px x 192px) you can replace the default avatar image with yours.